rocAL/rocAL/source/api/rocal_api_data_transfer.cpp

```diff
@@ -115,16 +115,26 @@ rocalCopyToOutput(
 }
 
 void
-ROCAL_API_CALL rocalSetOutputs(RocalContext p_context, unsigned int num_of_outputs, std::vector<RocalImage> &output_images)
+ROCAL_API_CALL rocalSetOutputs(RocalContext p_context, unsigned int num_of_outputs, std::vector<RocalTensor> &output_tensors)
 {
     if (!p_context)
         THROW("Invalid rocal context passed to rocalSetOutputs")
     auto context = static_cast<Context *>(p_context);
-    std::vector<Image*> output_images_vector ;
-    for (auto& it : output_images) {
-        auto img = static_cast<Image*>(it);
-        context->master_graph->set_output(img);
+    for (auto& it : output_tensors) {
+        auto tensor = static_cast<Tensor *>(it);
+        context->master_graph->set_output(tensor);
     }
 }
 
-
+RocalTensorList ROCAL_API_CALL
+rocalGetOutputTensors(RocalContext p_context) {
+    auto context = static_cast<Context*>(p_context);
+    try {
+        return context->master_graph->get_output_tensors();
```

Comment:
We have an api for GetOutputTensors, is it better if we add api named SetOutputTensors too?
I know setOutput does the job of setOutputTensors but since we have added an API for get, feel it might be better to add an api for set too

Reply:
Set already has this existing API, so It is better to have the name similar to that in TOT